src/db/models/userHomeAddress.js

```diff
@@ -0,0 +1,24 @@
+module.exports = (sequelize, DataTypes) => {
+  const UserHomeAddress = sequelize.define(
+    'UserHomeAddress',
+    {
+      line1: { allowNull: false, type: DataTypes.STRING },
```

Comment:
`street` and `streetNumber` instead of `line1`. The same should be valid to the rest of the columns.